docs/topics/wasm/wasm-overview.md

```diff
@@ -32,6 +32,16 @@ a compilation target to run on the web.
 Kotlin/Wasm compiles your Kotlin code into Wasm format. Using Kotlin/Wasm, you can create applications that run on 
 different environments and devices, which support Wasm and meet Kotlin's requirements. 
 
+Kotlin/Wasm benefits from [WebAssembly System Interface (WASI)](https://wasi.dev/), which provides 
```

Comment:
I think we need to reflect that Compose and WASI are two different sub-platforms and can't be used together. We have `wasmJs` Gradle target, where we can use Compose Multiplatform and run in browsers. As an alternative we have  a `wasmWasi` Gradle target, which is mostly intended for CLI and server-side apps, and where we can call WASI api and run it outside of browsers.

I think we can call these targets "Wasm-JS" and "Wasm-WASI" if we don't want to tie them too closely to Gradle here.

Given this distinction "Wasm-JS" doesn't "benefit from WASI" or "leverage WASI to abstract away platform-specific details", only "Wasm-WASI" does.

Reply:
This is great point. I restructured the overview so that it explains from general to specific. It starts with the information that is common to the 2 sub-platforms and contains 2 subsections about each sub-platform. Please check how it looks. 
ccp. @bashor